examples/DotNetCore/EventHub/SimpleAppConfigEventHub.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+      <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.1" />
+      <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.0.1" />
+      <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.3" />
+      <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.3" />
+      <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.3" />
+      <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+      <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="3.0.0" />
+      <PackageReference Include="Azure.Identity" Version="1.1.1" />
+      <PackageReference Include="Microsoft.Extensions.Configuration.AzureAppConfiguration" Version="3.0.0" />
```

Comment:
This will be brought in by `Microsoft.Azure.AppConfiguration.AspNetCore` already.